src/OrchardCore/OrchardCore.Localization.Abstractions/CultureAspect.cs

```diff
@@ -0,0 +1,23 @@
+using System.Globalization;
+
+namespace OrchardCore.Localization
+{
+    /// <summary>
+    /// Represents a culture metadata for the content item.
+    /// </summary>
+    public class CultureAspect
+    {
+        /// <summary>
+        /// Gets or sets the culture.
+        /// </summary>
+        public CultureInfo Culture { get; set; }
+
+        /// <summary>
+        /// Gets the language direction of the <see cref="Culture"/>.
+        /// </summary>
+        public string LanguageDirection
```

Comment:
Not sure if this is necessary as I think you can use CultureInfo.RTL

Reply:
Remove this property, but you can add an extension method on `CultureInfo` to get the direction as a string. And the static constants can be in the extension method's class.

Reply:
> I think you can use CultureInfo.RTL

There's no such property?!! AFAIK you can get it from ``TextInfo`

Reply:
> Remove this property, but you can add an extension method on CultureInfo to get the direction as a string. And the static constants can be in the extension method's class.

Same thing, but I find this clear